src/Controller/AuthenticationController.php

```diff
@@ -355,46 +389,29 @@ private function showUserIsBlockedErrorPage($isBlockedPermanently)
         return $this->forward(
             'App\Controller\ExceptionController::showAction',
             [
-                'exception'=> $exception,
+                'exception' => $exception,
             ]
         );
     }
 
     /**
-     * @param $notificationType
-     * @param $notificationAddress
-     * @return bool
+     * @param string $notificationType
+     * @param string $notificationAddress
+     * @return bool True when the notification was successfully sent, false otherwise
      */
-    private function sendNotification($notificationType, $notificationAddress)
+    private function sendNotification(string $notificationType, string $notificationAddress): bool
     {
-        $this->logger->notice(sprintf(
-            'Sending client notification for type "%s" and address "%s"',
-            $notificationType,
-            $notificationAddress
-        ));
-
-        // The GCM API has been replaced by FCM
-        if ($notificationType == 'GCM') {
-            $notificationType = 'FCM';
-
-            $this->logger->info(
-                sprintf(
-                    'Updating GCM notificationType to FCM',
-                    $notificationAddress
-                )
-            );
-        }
```

Comment:
:+1: :medal_military: Thanks for cleaning this up after the GCM grace period exceeded.